### PR TITLE
Update ABAP mission

### PR DESCRIPTION
**Pull request by mervey45**

### CODE REVIEW

File: tutorials/abap-environment-deploy-cf-production/abap-environment-deploy-cf-production.md [hunk 1]
```diff
@@ -371,87 +365,30 @@ author_profile: https://github.com/mervey45
 [ACCORDION-END]
 
 
-[ACCORDION-BEGIN [Step 9: ](Create index.html and run SAP Fiori application)]
+[ACCORDION-BEGIN [Step 9: ](Run SAP Fiori application)]
 
-  1. Go back to SAP Business Application Studio, select projects and open your project **`ztravel_app_xxx`**.Right-click your **`webapp`** folder and select **New File**.
+  1. Go back to SAP Business Application Studio and deploy your changes. Therefore right-click on your project **`ztravelappxxx`** and select **Open in Terminal**.
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-deploy-cf-production/abap-environment-deploy-cf-production.md:370]:
```suggestion
  1. Go back to SAP Business Application Studio and deploy your changes. Therefore right-click your project **`ztravelappxxx`** and select **Open in Terminal**.
```

File: tutorials/abap-environment-deploy-cf-production/abap-environment-deploy-cf-production.md [hunk 2]
```diff
@@ -371,87 +365,30 @@ author_profile: https://github.com/mervey45
...
-
-    ![url](url.png)
-
-
-  7.  Enter **`npm run deploy`**. When prompted, check deployment configuration and press y.
-
-      ![url](url2.png)
-
-  7. Press **`CTRL and click on the following link`** to open the URL in a browser.
+  3. Press **`CTRL and click on the following link`** to open the URL in a browser.
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-deploy-cf-production/abap-environment-deploy-cf-production.md:379]:
```suggestion
  3. Press **`CTRL and click the following link`** to open the URL in a browser.
```

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Please don't say "click on".

File: tutorials/abap-environment-deploy-cf-production/abap-environment-deploy-cf-production.md [hunk 3]
```diff
@@ -233,9 +227,9 @@ author_profile: https://github.com/mervey45
 
 [ACCORDION-BEGIN [Step 6: ](Deploy your application)]
 
-  1. Go back to SAP Business Application Studio, select projects, right-click on your project **`ztravel_app_xxx`** and select **Open in Terminal**.
+  1. Go back to SAP Business Application Studio, select projects, right-click on your project **`ztravelappxxx`** and select **Open in Terminal**.
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-deploy-cf-production/abap-environment-deploy-cf-production.md:230]:
```suggestion
  1. Go back to SAP Business Application Studio, select projects, right-click your project **`ztravelappxxx`** and select **Open in Terminal**.
```

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/abap-environment-deploy-cf-production/abap-environment-deploy-cf-production.md [hunk 4]
```diff
@@ -371,87 +365,30 @@ author_profile: https://github.com/mervey45
...
-
-      ![url](url2.png)
-
-  7. Press **`CTRL and click on the following link`** to open the URL in a browser.
+  3. Press **`CTRL and click the following link`** to open the URL in a browser.
 
       ![url](url3.png)
 
-  8. Login to ABAP Trial.
+  4. Login to ABAP Trial.
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-deploy-cf-production/abap-environment-deploy-cf-production.md:383]: "the login" or "to log in"?

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/abap-environment-deploy-cf-production/abap-environment-deploy-cf-production.md (hunk 4 above)
Comment (mervey45) [reply tutorials/abap-environment-deploy-cf-production/abap-environment-deploy-cf-production.md:383]: Log in

Comment (mervey45) [reply tutorials/abap-environment-deploy-cf-production/abap-environment-deploy-cf-production.md:383]: Logon to ABAP Trial.

**Pull request merged**